src/main/scripts/src/app/shared/perspectives/collections/post-it/post-it-collections-perspective.component.ts

```diff
@@ -117,19 +123,65 @@ export class PostItCollectionsPerspectiveComponent implements Perspective, OnIni
     }
   }
 
+  public hasWriteRole(collection: Collection) {
+    return collection.userRoles.indexOf(Role.write) !== -1;
+  }
+
+  public hasManageRole(collection: Collection) {
+    return collection.userRoles.indexOf(Role.manage) !== -1;
+  }
+
+  public onDetailClick(collectionCode: String) {
+    console.log('onDetailClick');
```

Comment:
Do we really want to log these? If this is currently being worked on, there should probably be `TODO` comment.